### PR TITLE
fix(cli): Add missing layer titles from the imagery to tileset.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**

### CODE REVIEW

File: packages/config/src/memory/memory.config.ts [hunk 1]
```diff
@@ -174,7 +174,7 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
         category: i.category,
         name: targetName,
         format: ImageFormat.Webp,
-        layers: [{ name: targetName, minZoom: 0, maxZoom: 32 }],
+        layers: [{ name: targetName, title: i.title, minZoom: 0, maxZoom: 32 }],
```
Comment (blacha) [inline packages/config/src/memory/memory.config.ts:177]: The All tileset should not be able to be viewed as a big layer, we could set the minZoom: 32?

File: packages/config/src/memory/memory.config.ts [hunk 2]
```diff
@@ -152,7 +152,7 @@ export class ConfigProviderMemory extends BasemapsConfigProvider {
     const allTileset: ConfigTileSet = {
       type: TileSetType.Raster,
       id: 'ts_all',
-      name: 'all_imagery',
+      name: 'all',
       title: 'All Imagery Basemaps',
```
Comment (blacha) [inline packages/config/src/memory/memory.config.ts:156]: Everything is a basemap? could this just be "All Imagery"

File: packages/config/src/memory/memory.config.ts (hunk 1 above)
Comment (Wentao-Kuang) [reply packages/config/src/memory/memory.config.ts:177]: Yes, this already been reset to zoom 32-32 in line 149 when creating all tileset.
This is creating individual tileset so it is zoom 0-32.

Comment (blacha) [reply packages/config/src/memory/memory.config.ts:177]: ahh the github diff showed it as the same block :face_exhaling:

**Review by blacha — APPROVED** [review]

**Pull request merged**